canonical-websites/build.snapcraft.io
# New branding

## Done

- Updates Vanilla to latest version (1.6.5) for consistent navigation styling with snapcraft.io
- Updates logo to a new branding
- Updates favicon to a new branding

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- Header navigation should have styles consistent with snapcraft.io and new logo
- Favicon should be a new logo


## Issue / Card

Fixes #251

## Screenshots

<img width="1240" alt="screen shot 2018-02-08 at 10 24 51" src="https://user-images.githubusercontent.com/83575/35965258-99c2e10c-0cba-11e8-9b81-d8282ad22620.png">
